src/open_inwoner/scss/components/Form/Checkbox.scss

```diff
@@ -1,4 +1,6 @@
 .checkbox {
+  padding-top: 1em;
```

Comment:
Not usually my preferred distance 🙂 and we do not wish to change ALL checkboxes in the project. 
So: do not add margins to a component that needs to be re-used; so: add class 'form' to the form in order to give it display grid - which gives the components inside a 'gap'